app/controllers/application_controller.rb

```diff
@@ -60,6 +60,23 @@ class RecordExists < Exceptions::RecordExists
 
   around_action :run_with_transaction
 
+  # path_default_content_type configures default content-types from within
```

Comment:
This is very informative, but I think you can convey enough information to be useful with a much shorter comment. The params are named such that they're fairly obvious and if I need an example I can grep for one in the code or look up the gem easily enough. Comment could be something like:

```
# sets the default content type header on incoming requests for controller methods
# that match the path_match regex
```